addons/website/views/snippets/snippets.xml

```diff
@@ -314,7 +318,7 @@
 
     <div data-js="sizing_y"
         data-selector="section, .row > div, .parallax, .s_hr, .carousel-item, .s_rating"
-        data-exclude="section:has(> .carousel), .o_gallery .carousel-item, .s_col_no_resize.row > div"/>
+        data-exclude="section:has(> .carousel), .o_gallery .carousel-item, .s_col_no_resize.row > div, .s_table_of_content_navbar_wrap"/>
```

Comment:
Why has the s_table_of_content_navbar_wrap element the s_col_no_resize class ? Can't we allow resizing the main columns ?

Reply:
As discussed, I changed it to allow resizing